packages/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+
+import CrownSolid from '@igloo-ui/icons/dist/CrownSolid';
+import TextBulletSolid from '@igloo-ui/icons/dist/TextBulletSolid';
+
+import './tabs.scss';
+
+export interface TabInterface {
+  /** The tab pane contents of the tabs list */
+  children?: React.ReactNode;
+  /** Add an id attribute to the tab */
+  id?: string;
+  /** Add text for the tab */
+  label: React.ReactNode;
+  /** Add a notification indicator to the tab */
+  notification?: boolean;
+  /** Add a premium indicator to the tab */
+  premium?: boolean;
+}
+
+export interface TabsProps extends React.ComponentProps<'div'> {
+  /** The function that's called when a tab is clicked */
+  onSelectTab?: (index: number) => void;
+  /** The index number of the Tab that should be selected on load */
+  selected?: number;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** Add a specific class to the Tabs component */
+  className?: string;
+  /** Add an inline style to the Tabs */
+  isInline?: boolean;
+  /** An array of objects that specify the props of the tab buttons */
+  tabs: Array<TabInterface>;
+}
+
+const Tabs: React.FunctionComponent<TabsProps> = ({
+  onSelectTab,
+  selected = 0,
+  dataTest,
+  className,
+  isInline = true,
+  tabs,
+}) => {
+  const classes = cx('ids-tabs', className, {
+    'ids-tabs--inline': isInline,
+    'ids-tabs--heading': !isInline,
+  });
+
+  /* Handle the on click */
```

Comment:
I don't think we need a comment here, the constant name already define what it is good enough.

Reply:
Sometimes I like to put comments so it's easier to find certain things when scrolling through the component, but if you want I can remove them.

Reply:
My thinking with comments is that it's always obsolete once you write it. You never know when someone else will change/refactor something that will make a comment tell a lie.

So overall, I prefer to set the less comment possible and only have one if it's impossible to easily do it directly in the code.

Some comments I think we can accept:
- For `REGEX`
- For complex `switch-case`
- Sometime, to define sections

Reply:
makes sense. I have removed them.